### ISSUE TITLE
Does not produce MSI file.

### ISSUE THREAD

**Issue by markmorris** [electron/windows-installer#271]
Does not produce an .msi installer regardless of config or settings.

Using version 2.7.0.

```
let settings = {
    name: ID,
    title: appName+' Sing It',
    appDirectory: path.join(outPath, ID+'-win32-ia32/'),
    authors: 'Musicline School Musicals',
    outputDirectory: path.join(outPath, 'windows-installer'),
    exe: appName+' Sing It.exe',
    setupExe: appName+' install.exe',
    setupMsi: appName+' install.msi',
    iconUrl: 'https://singit.musiclinedirect.com/_admin/_data/icon.ico',
    setupIcon: path.join(rootPath, 'assets', 'icons', 'windows', 'icon.ico'),
    loadingGif: path.join(rootPath, 'assets', 'images', 'Artboard.gif')
};
```

Theres no error creates the normal .exe installer perfectly and runs fine. Just no .msi

**Comment by markmorris** [comment]
UPDATE: 
I've seen another issue in regards to no .msi with dashes or spaces in the main .exe, just want to say i have tried this with a single word in the .exe and it still produces nothing with no errors.

**Comment by MaxCloutier** [comment]
I also have this problem, any solution/fix/hack available in the meantime?

**Comment by markmorris** [comment]
If you are using Linux theres no hope. Unless you can get wine and WixToolset to work correctly.

**Comment by MaxCloutier** [comment]
Yeah I'm on mac, the only solution I found was to get a third party solution to build an MSI from my exe on a windows computer... Not the prettiest solution, but it'll do for now. Thanks for the reply!

**Comment by YeetOrBeYate** [comment]
> Yeah I'm on mac, the only solution I found was to get a third party solution to build an MSI from my exe on a windows computer... Not the prettiest solution, but it'll do for now. Thanks for the reply!

@MaxCloutier Which thrid party solution did you end up going with if you don't mind me asking? I'm stuck trying to make a MSI file with no success on my own.

**Comment by MaxCloutier** [comment]
> > Yeah I'm on mac, the only solution I found was to get a third party solution to build an MSI from my exe on a windows computer... Not the prettiest solution, but it'll do for now. Thanks for the reply!
> 
> @MaxCloutier Which thrid party solution did you end up going with if you don't mind me asking? I'm stuck trying to make a MSI file with no success on my own.

I don't remember unfortunately, I moved on to a different company unfortunately and believe the company I did this for isn't using it anymore. It's been a while since then so I would assume that there might be better solutions out there now :) I wish you the best in finding a solution, I believe I had to have a program running on windows, sending my app from mac to windows, then converting it to a MSI file. There has to be a better solution out there.
